modules/lib/maintainers.nix

```diff
@@ -502,6 +502,12 @@
     github = "shimunn";
     githubId = 41011289;
   };
+  rapiteanu = {
+    name = "Viorel-Cătălin Răpițeanu";
+    email = "rapiteanu.catalin@gmail.com";
+    github = "Steinhagen";
+    githubId = 4029937;
+  };
```

Comment:
Since you are already in the Nixpkgs maintainers list you don't need to also add yourself here.
```suggestion
```

Reply:
I will remove that.